Commands/Public/room.js

```diff
@@ -0,0 +1,195 @@
+const ArgParser = require("../../Modules/MessageUtils/Parser");
+const moment = require("moment");
+
+module.exports = async ({ Constants: { Colors, Text }, client }, { serverDocument, serverQueryDocument }, msg, commandData) => {
+	const roomDocument = serverDocument.config.room_data.id(msg.channel.id);
+	if (roomDocument) {
+		const question = await msg.send({
+			embed: {
+				color: Colors.PROMPT,
+				description: `This room was created ${moment(roomDocument.created_timestamp).fromNow()}. Would you like to delete it?`,
+			},
+			footer: {
+				text: "You have 1 minute to respond.",
+			},
+		});
+		let response = (await msg.channel.awaitMessages(res => res.author.id === msg.author.id, { max: 1, time: 60000 })).first();
+		if (response) {
+			try {
+				await response.delete();
+			} catch (err) {
+				// No-op
+			}
+		}
+		if (response && configJS.yesStrings.includes(response.content.toLowerCase().trim())) {
+			let success = true;
+			await msg.channel.delete(`Room Management | Command issued by ${msg.author.tag}`).catch(err => {
+				success = false;
+				winston.debug(`Failed to delete room '${msg.channel.name}' on server '${msg.guild.name}'`, { svrid: msg.guild.id, chid: msg.channel.id, err });
+				msg.send({
+					embed: {
+						color: Colors.LIGHT_ERR,
+						title: "Failed to delete room!",
```

Comment:
```suggestion
						title: "Failed to delete the room!",
```

Reply:
I disagree, I feel the sentence works better without the article.

Reply:
No? It sounds like someone doesn't know english. Say it out loud 4 times, and thank me later

Reply:
Sorry, but `the room` sounds absolutely awful.